builder/virtualbox/ovf/config.go

```diff
@@ -87,13 +89,12 @@ func NewConfig(raws ...interface{}) (*Config, []string, error) {
 	errs = packer.MultiErrorAppend(errs, c.VBoxManagePostConfig.Prepare(&c.ctx)...)
 	errs = packer.MultiErrorAppend(errs, c.VBoxVersionConfig.Prepare(&c.ctx)...)
 
-	if c.SourcePath == "" {
-		errs = packer.MultiErrorAppend(errs, fmt.Errorf("source_path is required"))
```

Comment:
sorry, just getting back to this and forgetting some context. Is there a case where we might not have a sourcepath? It seems like we should still error here